packages/itwinui-react/src/core/Typography/Anchor.tsx

```diff
@@ -18,4 +32,24 @@ import { polymorphic } from '../utils/index.js';
  * <Anchor as={Link} to='/'>Home</Anchor>
  * <Anchor as='button' onClick={() => {}}>click me</Anchor>
  */
-export const Anchor = polymorphic.a('iui-anchor');
+export const Anchor = React.forwardRef((props, forwardedRef) => {
+  const { isExternal, children, ...rest } = props;
+
+  return (
+    <Box
+      as='a'
+      {...rest}
+      ref={forwardedRef}
+      className={cx(
+        'iui-anchor',
+        { 'iui-anchor-external': isExternal },
+        props.className,
+      )}
+    >
+      {children}
+      {props.target === '_blank' && (
```

Comment:
non-blocking: Maybe later we can handle more values of the target prop ([docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target#usage_notes))

Reply:
The other values of `target` are rarely used, and also do not require any special handling that I can think of.